examples/coap/src/bin/coapclient.rs

```diff
@@ -53,7 +53,7 @@ fn client_handshake() -> Result<(), EDHOCError> {
 
     let mut msg_3 = Vec::from(c_r.as_cbor());
     let (mut initiator, message_3, prk_out) =
-        initiator.prepare_message_3(CredentialTransfer::ByReference, &None)?;
+        initiator.prepare_message_3(CredentialTransfer::ByValue, &None)?;
```

Comment:
Do we want to change this here or is this just an artifact of debugging?

Reply:
Accepted.